src/compiler/transformers/ts.ts

```diff
@@ -2455,9 +2455,11 @@ namespace ts {
         }
 
         function shouldEmitAliasDeclaration(node: Node): boolean {
-            return compilerOptions.preserveValueImports
-                ? resolver.isValueAliasDeclaration(node)
-                : resolver.isReferencedAliasDeclaration(node);
+            return isInJSFile(node)

```

Comment:
Is this just `||`?

Reply:
yep, but I guess I wanted to keep using ternaries since that was already there. I'll try to fix.